templates/decl.j2

```diff
@@ -32,6 +32,29 @@
 
 {{ decl.doc_string | convert_markdown }}
 
+{% if decl.informal_statement %}
+
+<details open>
+    <summary>Informal translation</summary>
+
+    <div class="informal_statement">
+        <div class="statement">
+            {{ decl.informal_statement }}
```

Comment:
```suggestion
            {{ decl.informal_statement | convert_markdown }}
```
The escaping code is in `convert_markdown` so this might fix things? Not tested though...